cypress/e2e/pages/batches.pages.js

```diff
@@ -62,7 +62,8 @@ function executeTransaction() {
 }
 
 function addToBatchButton() {
-  cy.contains(addToBatchBtn).should('be.visible').and('not.be.disabled').click()
+  cy.wait(1000)
+  cy.get('button').contains(addToBatchBtn).scrollIntoView().should('not.be.disabled').click()
```

Comment:
Here you don't check if the button is visible, so no need to scroll to it either.